codegen/test/chart/templates/deployment.yaml

```diff
@@ -120,39 +122,45 @@ spec:
       imagePullSecrets:
         - name: {{ $painterImage.pullSecret }}
       {{- end}}
-
+{{- end }}
```

Comment:
i think we can remove the enabled check from the definition since we do the check around where we call the definiton, wdyt?